src/components/profileEdit/AddressInfo.tsx

```
@@ -1,142 +1,147 @@
+    const handleClickSetDefaultButton = (accountId: number) => {
+        setDefaultId(accountId)
+        setIsDefaultModalOpen(true)
+    }
+
+    useEffect(() => {
+        loadData()
+    }, [])
+
     return (
         <div className="flex gap-4">
+            <ConfirmModal isOpen={isDefaultModalOpen} message={"기본 계좌로 설정할까요?"} onConfirm={setDefaultDelivery} onClose={() => setIsDefaultModalOpen(false)} />
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: loadData

This dependency is not specified in the hook dependency list.

Either include it or remove the dependency array

(lint/correctness/useExhaustiveDependencies)